test/InterfacesTest.ts

```diff
@@ -1,7 +1,7 @@
 // Test typescript interfaces for ts compliation errors
-import * as azf from "../types/public/Interfaces";
+import { IFunction, IContext, IRequest} from "../types/public/Interfaces";
 
```

Comment:
changes to this file aren't really relevant, just trying a different (more common) way of importing and rearranged order of functions